File: messaging/v1/alpha_sender/read-service-alpha-list/read-service-alpha-list.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Messaging.V1.Service;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var alphaSenders = AlphaSenderResource.Read(
            pathServiceSid: "MG2172dd2db502e20dd981ef0d67850e1a"
        );

        foreach(var record in alphaSenders)
        {
           Console.WriteLine(record.Sid);
        }
    }
}

File: messaging/v1/alpha_sender/read-service-alpha-list/read-service-alpha-list.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

alpha_senders = @client.messaging
                       .services('MG2172dd2db502e20dd981ef0d67850e1a')
                       .alpha_senders
                       .list

alpha_senders.each do |record|
  puts record.sid
end
